bin/generate

```diff
@@ -42,17 +42,13 @@ if [[ -z ${HP_MASTERNODES_AMD_COUNT} ]] || [[ -z ${HP_MASTERNODES_ARM_COUNT} ]];
 fi
 
 SEEDS_COUNT="1"
-MIXER_COUNT="0"
```

Comment:
Why did we remove mixer count here?

Reply:
As discussed, generate is being made easier to use. Another PR will go in after this one to pull values from tfvars